scripts/thanks.js

```diff
@@ -0,0 +1,33 @@
+if (!remoteUrl) {
+  var remoteUrl = "https://api.carpoolvote.com/live";
+}
+
+const handleBackendResponse = () => {
+  var data = tinyQuery.getAll();
+
+  var heading             = document.getElementById("thanks-header");
+  var introPara           = document.getElementById("introPara");
+  var successInfo         = document.getElementById("requestSuccessInfo");
+  var responseDetails     = document.getElementById("responseDetails");
+  var responseCode        = document.getElementById("responseCode");
+  var responseText        = document.getElementById("responseText");
+
+  var headingFailureText  = document.getElementById("sorryHeading").innerHTML;
+  var paraFailureText     = document.getElementById("sorryPara").innerHTML;
+
+  if (data.code != "0") {
+    heading.innerHTML   = headingFailureText;
+    introPara.innerHTML = paraFailureText;
+
+    successInfo.setAttribute("hidden", "hidden");
+    successInfo.setAttribute("style", "display:none");    
+
+    responseDetails.removeAttribute("hidden");
+    responseDetails.setAttribute("style", "");    
+
+    responseCode.innerHTML = data.code;
+    responseText.innerHTML = data.info;
+  }
+} 
```

Comment:
No linter, I just mean in general, it's considered proper JS formatting to include a semicolon after an function expression. I wouldn't hold up a PR over it, I just figured I'd point it out while I'm going over it. If you prefer not to use semicolons for this sort of thing then it's not a big deal, don't worry about it.